Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -13785,7 +13796,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Your **@username** is your identity in the Nos community.\u2028\nChoose a name that reflects you or your organization. Make it memorable and distinct!"
+            "value" : "Your **@username** is your identity in the Nos community. \nChoose a name that reflects you or your organization. Make it memorable and distinct!"
```

Comment:
I didn't change this, I wonder why it keeps showing up.

Reply:
Xcode removes the `\u2028` from strings whenever you edit Localizable.xcstrings. It's annoying but I'm not sure what we can do about it.

Reply:
And Crowdin keeps adding these back. I have an email thread going with Crowdin trying to figure this out. For now it's fine to leave it in.

Reply:
🤦‍♀️